### ISSUE TITLE
ERROR: Can't access the Fragment View's LifecycleOwner when getView() is null i.e., before onCreateView() or after onDestroyView()

### ISSUE THREAD

**Issue by donenoyes** [KaustubhPatange/Moviesy#20]
**Describe the Bug:**

```
2021-04-02 16:58:20.916 20051-20051/com.kpstv.yts E/CrashOnActivity: Application crashed
    java.lang.IllegalStateException: Can't access the Fragment View's LifecycleOwner when getView() is null i.e., before onCreateView() or after onDestroyView()
        at androidx.fragment.app.Fragment.getViewLifecycleOwner(Fragment.java:361)
        at com.kpstv.common_moviesy.extensions.FragmentViewBindingDelegate.getValue(ViewBindings.kt:71)
        at com.kpstv.yts.ui.fragments.LibraryFragment.getBinding(Unknown Source:10)
        at com.kpstv.yts.ui.fragments.LibraryFragment.onReselected(LibraryFragment.kt:80)
        at com.kpstv.yts.ui.activities.AbstractBottomNavActivity$onStart$1.invoke(AbstractBottomNavActivity.kt:89)
        at com.kpstv.yts.ui.activities.AbstractBottomNavActivity$onStart$1.onNavigationItemSelected(AbstractBottomNavActivity.kt:22)
        at com.google.android.material.bottomnavigation.BottomNavigationView$1.onMenuItemSelected(BottomNavigationView.java:243)
```

**Expected Behavior:**

App to not crash.

**Comment by KaustubhPatange** [comment]
I've been seeing such similar issues on Crashlytics as well. It seems like there are some unexpected fragment state behaviours when navigating through `bottomNav`.

Just one question before you navigate to Library fragment did the application was in background for quite some time?

**Comment by KaustubhPatange** [comment]
I found out this issue happens when there are some recursive clicks on `BottomNavBar`. This commit 913ad1c fixes it.

**Comment by KaustubhPatange** [comment]
I have released the v1.6 version, this has been fixed. See the complete changelog [here](https://github.com/KaustubhPatange/Moviesy/releases/tag/v1.6)
